RemoteSignTool/RemoteSignTool.Server/Services/SignToolService.cs

```diff
@@ -26,16 +27,28 @@ public bool TryToFindSignToolPath(out string path)
                 path = SignToolX86Path;
                 return true;
             }
-            else if (File.Exists(SignTool_15063_X64Path))
-            {
-                path = SignTool_15063_X64Path;
-                return true;
-            }
             else
             {
-                path = null;
-                return false;
+                System.IO.DirectoryInfo sdkRoot = new System.IO.DirectoryInfo(WindowsSDKRootPath);
```

Comment:
I believe we can skip full namespace when using classes from System.IO, it'll be easier to read the code.